r/R/feather.R

```diff
@@ -17,8 +17,8 @@
 
 #' Write data in the Feather format
 #'
-#' @param x `data.frame`, `RecordBatch`, or `Table`
-#' @param sink A file path or an `OutputStream`
+#' @param x `data.frame`, [RecordBatch], or [Table]
+#' @param sink A string file path, [OutputStream], or [RecordBatchWriter]
```

Comment:
This doesn't seem right to me. RecordBatchWriter a generic interface and cannot be guaranteed to produce something that can be read by `read_feather`. For example, it may write an IPC stream. Flight also has implementations of RecordBatchWriter and those do something completely different